docs/version0.38.md

```diff
@@ -46,6 +47,12 @@ This option controls the handling of the `SIGUSR2` signal by the VM signal handl
 
 For more information, see [`-XX:[+|-]HandleSIGUSR2`](xxhandlesigusr2.md).
 
+### Technical preview of CRIU feature
+
+A technical preview of CRIU feature is included in this release. It is currently available for OpenJDK 11 and later running on Linux&reg; x86, Linux on POWER LE&reg;, and Linux on IBM Z&reg; only.
+
+Checkpoint Restore in User Space (CRIU) feature can be used to stop the VM at a particular checkpoint, save the state, and then enable the VM to resume from the point where it was stopped. You can restore the checkpoint image in as many environments as possible with improved start-up time and hence with better VM performance. For more information, see [CRIU feature](criufeature.md).
```

Comment:
This sentence should be active voice not passive, ie "you can use" not "can be used". "particular" is a bit redundant. "save the state" should probably be "save its state", ie the VM's state. "enable the VM to resume" can probably be replaced with "run". ie "You can use the CRIU feature to stop the VM at a checkpoint, save its state, then run the VM from the point where it was stopped."

Reply:
Done